galley/pkg/config/processor/transforms/serviceentry/converter/instance.go

```diff
@@ -48,8 +48,8 @@ func New(domain string, pods pod.Cache) *Instance {
 // ServiceEntry is passed as an argument (out) in order to enable object reuse in the future.
 func (i *Instance) Convert(service *resource.Entry, endpoints *resource.Entry, outMeta *resource.Metadata,
 	out *networking.ServiceEntry) error {
-	i.convertService(service, outMeta, out)
 	i.convertEndpoints(endpoints, outMeta, out)
+	i.convertService(service, outMeta, out)
```

Comment:
can you add a comment that order is important?

Reply:
Sure, basically we want to build the endpoints first as it would have an impact on determining the service resolution e.g. Static service resolution must have endpoints. I will add the comment.